cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -390,3 +399,13 @@ func (k *Kuberhealthy) getCurrentState() (health.State, error) {
 	}
 	return state, nil
 }
+
+// getCheck returns a Kuberhealthy check object from its name, returns an error otherwise
+func (k *Kuberhealthy) getCheck(name string) (KuberhealthyCheck, error) {
+	for _, c := range k.Checks {
+		if c.Name() == name {
+			return c, nil
+		}
+	}
+	return nil, fmt.Errorf("Could not find Kuberhealthy check with name %s", name)
```

Comment:
This project uses the logrus package for logging output.

Reply:
fmt.Errorf doesn't output, it returns an error